src/taipy/templates/multi-page-gui/{{cookiecutter.application_name}}/pages/root/root.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from taipy.gui import Markdown
+
+root_md = Markdown("pages/root/root.md")
```

Comment:
@FabienLelaquais @FlorianJacta Don't you think we can add some comments in the file to explain the root page purpose?

Reply:
Same comment as before: everything generated should or might have some explanation.
The root page in my option should not be in a (useless) subdirectory.

Reply:
I add some intro string to the generated python files. Please take a look.

If it's ok, I will update the default template as well.